app/controllers/users_controller.rb

```diff
@@ -1,10 +1,21 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:edit, :update, :index]
+  before_action :correct_user, only: [:edit, :update]
+  before_action :is_admin, only: :destroy
+  before_action :load_user, except: [:index, :new, :create]
+
+  def index
+    @users = User.select("id, name, email").order("name asc").paginate page: params[:page], per_page: Settings.per_page
```

Comment:
* `select(:id, :name, :email)`
* `order(name: :asc)`
* not more than 80 characters per line